packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
+ *    .setItemsPerPage(2)
+ *    .make()
+ *    .run(message.author, message.channel);
+ * ```
+ */
+export class PaginationFieldMessageEmbed<Element> extends PaginatedMessage {
+	private embedTemplate: MessageEmbed = new MessageEmbed();
+	private totalPages = 0;
+	private items: Element[] = [];
+	private itemsPerPage = 10;
+	private fieldTitle!: string;
+
+	/**
+	 * Items
+	 *
+	 * Array of items for pagination.
+	 * @param items The pages to set
+	 */
+	public setItems(items: Element[]) {
+		this.items = items;
+		return this;
+	}
+
+	/**
+	 * Title
+	 *
+	 * Title of the embed field that will be used to paginate the items.
+	 * @param title Field title
+	 */
+	public setTitleField(title: string) {
+		this.fieldTitle = title;
+		return this;
+	}
+
+	/**
+	 * items per page
+	 * Number of items to show per page.
+	 * @param itemsPerPage Number o items
+	 */
+	public setItemsPerPage(itemsPerPage: number) {
+		this.itemsPerPage = itemsPerPage;
+		return this;
+	}
+
+	/**
+	 * Embed template
+	 *
+	 * Template to be used to display the pagination of the articles. This template can be configured by the `new MessageEmbed()` class or directly by means of an object with embed options.
+	 *
+	 * @example
+	 * ```typescript
+	 * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+	 * const { MessageEmbed } = require('discord.js');
+	 *
+	 * // Option 1:
+	 * new PaginationFieldManager().setTemplate(new MessageEmbed().setTitle('Test pager embed')).make().run(message.author, message.channel);
+	 * // Option 2:
+	 * new PaginationFieldManager().setTemplate({ title: 'Test pager embed' }).make().run(message.author, message.channel);
+	 * ```
+	 * @param template MessageEmbed
+	 */
+	public setTemplate(template: MessageEmbedOptions | MessageEmbed) {
+		this.embedTemplate = new MessageEmbed(template);
+		return this;
+	}
+
+	/**
+	 * Format field
+	 *
+	 * The way in which the items of the object array will be represented in the embed field to be paginated.
```

Comment:
This sentence makes no sense to me at all...

Reply:
It means that the items will be represented in the way that is configured here, it is the same as mapping an array.

For example:
```ts
const array = [{ { field: '1', value: '2' }, { field: '1', value: '2' }];
array.map((item) => `${item.field} ${item.value}` );
```

Return the item data as a string to be repressed in the embed.